src/Marten/Linq/Fields/DuplicatedField.cs

```diff
@@ -89,8 +89,8 @@ public class DuplicatedField: IField
         /// </summary>
         public NpgsqlDbType DbType { get; set; }
 
-
-        internal UpsertArgument UpsertArgument => new UpsertArgument
+        /// <inheritdoc />
+        public UpsertArgument ToArgument() => new UpsertArgument
```

Comment:
**NOTE:** This is a breaking change.

Reply:
It's a breaking change but it was internal API. 
I wanted to unify all objects that provide UpsertArgument. IFunctionArgument will be used instead of UpsertArgument